.github/workflows/build-artifact.yml

```diff
@@ -0,0 +1,59 @@
+name: Build artifact
+
+on:
+  push:
+    paths:
+      - 'plugin/**'
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+    paths:
+      - 'plugin/**'
+
+jobs:
+  build:
+    runs-on: windows-latest
+
+    strategy:
```

Comment:
I don't necessarily think it matters, but I don't think you need this to have a testing matrix; I think you could specify the job to run with setup-python at your preferred version and run that way. I don't foresee a scenario where we will test the MSBuild against multiple versions of Python

Reply:
I see.
So this option is used for testing python program in different python environments, since this project only needs one runnable python environment, we don't need a matrix.